### PR TITLE
Updated dependency versions

### PR DESCRIPTION
**Pull request by AvinashChowdary**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,10 @@ all languages.
 
 ## Release Notes
 
+### Release 2.0.2 (June 3, 2021)
+* Updated to version 2.3.4 of the Amazon Kinesis Client Library for Java.
+* Upgraded multiple dependencies
```
Comment (joshua-kim) [inline README.md:149]: We should make release notes / dependency upgrades separate PRS, and we should link PRS once they're merged in so customers can reference them in release notes.

Also, we need to get these release notes approved by doc writer.

**Comment by joshua-kim** [comment]
How did we test this?

File: README.md (hunk 1 above)
Comment (joshua-kim) [reply README.md:149]: Also good to list what dependencies were upgraded specifically (i.e AWS SDK, Jackson, etc.)

**Review by joshua-kim — CHANGES_REQUESTED** [review]
Left a few comments.

**Comment by AvinashChowdary** [comment]
Tested by running the sample https://github.com/awslabs/amazon-kinesis-client-python#running-the-sample. Was able to successfully write and read records from Kinesis Stream.

**Review by joshua-kim — APPROVED** [review]
LGTM

**Pull request merged**